src/control/cmd/daos/util.go

```diff
@@ -152,33 +150,24 @@ func createWriteStream(ctx context.Context, prefix string, printLn func(line str
 		return nil, nil, err
 	}
 
-	go func(ctx context.Context, prefix string) {
-		if prefix != "" {
-			prefix = ": "
-		}
+	go func(ctx context.Context) {
 
 		rdr := bufio.NewReader(r)
 		for {
-			select {
-			case <-ctx.Done():
-				return
```

Comment:
I'm not sure about removing this. Is this what's cutting off the output? The purpose of these lines is to exit this goroutine in the event that the user exits the program.

Reply:
Will try re-adding, thank you for the review.